docs/UserGuide.md

```diff
@@ -218,88 +230,94 @@ designers, and boutique flower businesses.
 
 </div>
 
-### Viewing help : `help`
+### 6.1. Viewing help : `help`
 
 Shows a message explaning how to access the help page.
 
 ![help message](images/helpMessage.png)
 Format: `help`
 
-### Clearing all entries : `clear`
+### 6.2. Clearing all entries : `clear`
 
-Clears all entries from the address book.
+Clears all entries from the BookKeeper.
 
 Format: `clear`
 
+### 6.3. Exiting the program : `exit`
+
+Exits the program.
+
+Format: `exit`
+
 ### <ins>Client Features:
 
-### Adding a person: `add`
+### 6.4. Adding a client: `add`
 
-Adds a person to the address book.
+Adds a client to the BookKeeper.
```

Comment:
We might want to omit the `the`. As above

Reply:
done